index.html

```diff
@@ -15,8 +15,8 @@
           <h1>Christina Truong</h1>
           <h2>Tech Educator + Front-end Developer</h2>
 
-          <p>As a develper, I specialize in creting modular and scalable front-end architectures. As an educator, I focus on creating inclusiv learning environments, instructor training and curriculum develoment.</p>
-          <p>I’m also exploring more creative pursuits designing tee shirts and accessories for <a href="http://teethang.com" target="_blank">Nuthin’ But a Tee Thang</a>, an online store Im running with my husband.</p>
+          <p>As a developer, I specialize in creating modular and scalable front-end architectures. As an educator, I focus on creating inclusiv learning environments, instructor training and curriculum develoment.</p>
```

Comment:
inclusiv = inclusive
develoment = development

Reply:
Disse to er stadig ikke rettet